test/atm/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 # tests that can run without needing all JEDI components
 
 # tests that require the full build
-if(BUILD_GDASBUNDLE)
+if (${BUILD_GDASBUNDLE})
```

Comment:
Is this necessary?  I thought this meant it was using an env var versus a CMake var? But perhaps BUILD_GDASBUNDLE is not properly set? Or I'm mistaken

Reply:
I was just following the pattern in `test/CMakeLists.txt`.   It contains `if (${BUILD_GDASBUNDLE})` in a few places.  Let me remove `${ }`, rebuild, and see what happens.

Reply:
Removing `${ }` didn't do any harm.   ATM 3DVAR ctests still run.  I'll commit this change to `feature/ctest_atm_3dvar` after I work through all your feedback.